README.md

```diff
@@ -24,26 +27,38 @@ The available Go binaries
 ## Using the Test Splitter
 
 ### ENV variables
+Test Splitter uses the following environment variables provided in your Buildkite build environment. 
+Please make sure that the following environment variables are available in your testing environment, particularly if you use Docker or some other type of containerization to run your tests.
+| Environment Variable | Description|
+| -------------------- | ----------- |
+| `BUILDKITE_BUILD_ID` | The UUID of the Buildkite build. Test Splitter uses this UUID along with `BUILDKITE_STEP_ID` to uniquely identify the test plan. |
+| `BUILDKITE_JOB_ID` | The UUID of the job in Buildkite build. |
+| `BUILDKITE_ORGANIZATION_SLUG` | The slug of your Buildkite organization. |
+| `BUILDKITE_PARALLEL_JOB` | The index of parallel job created from a Buildkite parallel build step. Test Splitter uses this value to get specific test plan for each parallel job. <br>Make sure you configure `parallelism` for your Buildkite build step.  You can read more on Buildkite parallel build step on this [page](https://buildkite.com/docs/pipelines/controlling-concurrency#concurrency-and-parallelism).| 
+| `BUILDKITE_PARALLEL_JOB_COUNT` | The total number of parallel jobs created from a Buildkite parallel build step. Test Splitter uses this value to split the tests and create the test plan. <br>Make sure you configure `parallelism` for your Buildkite build step.  You can read more on Buildkite parallel build step on this [page](https://buildkite.com/docs/pipelines/controlling-concurrency#concurrency-and-parallelism) |
+| `BUILDKITE_STEP_ID` | The UUID of the step group in Buildkite build. Test Splitter uses this UUID along with `BUILDKITE_BUILD_ID` to uniquely identify the test plan.
```

Comment:
I understand that you've ordered this list alphabetically, but I think it might make more sense to group them semantically? i.e. 
BUILDKITE_BUILD_ID
BUILDKITE_STEP_ID
BUILDKITE_ORGANIZATION_SLUG
BUILDKITE_PARALLEL_JOB_COUNT
BUILDKITE_PARALLEL_JOB

And then you won't have to repeat the explanations all the time?

Reply:
I'll keep it sorted alphabetically to be consistent with other table and a practice we do in our [docs](https://buildkite.com/docs/pipelines/environment-variables).

We will have to repeat the explanations anyway since it is a table format. We will have to change the format, or merge some rows to have a single description.